src/accounts/views.py

```diff
@@ -69,6 +69,29 @@ def change(request):
         form = UserChangeForm(request.POST, instance=request.user)
         if form.is_valid():
             form.save()
+            #to get the value of the 'attestationEmails' checkbox.
+            attestationEmails = request.POST.get('attestationEmails')
+            #to change the values of the value of the checkbox to 'True' or 'False'
+            is_checked = bool(attestationEmails)
```

Comment:
this is never used

Reply:
the comment should relate to line 73-88. I would guess

            user = form.save(commit=False)
            attestationEmails = form.cleaned_data['attestationEmails']
            uploadConfirmEmails = form.cleaned_data['uploadConfirmEmails']
            user.attestationEmails = attestationEmails
            user.uploadConfirmEmails = uploadConfirmEmails
            user.save()

does the job

Reply:
@Abdallah-Abouelatta I mean that only the lines quoted above should remain; the rest assigns a variable that is never used and can be deleted.